include/items.php

```diff
@@ -2240,7 +2241,7 @@ function drop_item($id, $interactive = true) {
 		}
 
 		// If item has attachments, drop them
-		foreach (explode(", ", $item['attach']) as $attach) {
```

Comment:
Just thinking, would not be one space too much in the $matches now, that the explode is not including the space after the comme?